tests/SConscript

```diff
@@ -1,4 +1,5 @@
 # -*- python -*-
 from lsst.sconsUtils import scripts
 scripts.BasicSConscript.pybind11(['testLib'])
-scripts.BasicSConscript.tests(noBuildList=['testLib.cc'])
```

Comment:
Pybind11 now builds it, so it needs to be disabled from the list of things that the test system has to build.